tests/rules/test_WildcardResourceRule.py

```diff
@@ -668,222 +655,13 @@ def test_multiple_resources_with_wildcard_resources_are_detected(user_and_policy
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:ReceiveMessage"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:RemovePermission"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:SendMessage"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:SendMessageBatch"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:SetQueueAttributes"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:TagQueue"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-            Failure(
-                granularity="ACTION",
-                reason='"RolePolicy" is using a wildcard resource for "sqs:UntagQueue"',
-                risk_value="MEDIUM",
-                rule="WildcardResourceRule",
-                rule_mode="BLOCKING",
-                actions={"sqs:*"},
-                resource_ids={"RolePolicy"},
-            ),
-        ],
-    )
-
+def test_policy_document_with_wildcard_resource_without_policy_name_is_detected():
```

Comment:
We should fix the policy, but we still need the test

Reply:
As mentioned in this commit message https://github.com/Skyscanner/cfripper/pull/179/commits/5a62065c5936cc94e704e73ea29b530739ee46c3 – you already do.

Look at [the test above](https://github.com/Skyscanner/cfripper/pull/179/commits/5a62065c5936cc94e704e73ea29b530739ee46c3#diff-6a6e3405779c9ba39d22e4ad625d916c7375cebd52f27d51cda9272f6d95ce11R97) or better yet do a diff between the file in this test and the one above. You already have the test for a template with a 'PolicyName' field. It is covered already.

I only kept this one, and the commit before to show that the template is malformed and it is picked up by pydantic type checks 👍